src/components/VersionRestartModal/VersionRestartModal.tsx

```diff
@@ -208,4 +303,15 @@ const Row = styled.div`
   }
 `;
 
+export const TitleContainer = styled("div")`
+  margin-top: 15px;
+  width: 96%;
+`;
+
+const BoldTextStyle = styled("div")`
```

Comment:
`styled.div`

Reply:
For html elements we have been using the dot notation for emotion styling elements.